docs/sources/write/links/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Links
+description: Understand how to link between pages.
+weight: 600
+aliases:
+  - /docs/writers-toolkit/write/links/
+  - /docs/writers-toolkit/write/references/
+  - /docs/writers-toolkit/writing-guide/references/
+keywords:
+  - Hugo
+  - link
+  - linking
+  - links
+  - ref
+  - references
+  - relref
+---
+
+# Links
+
+Choose your link type based on the applicable scenario:
+
+- [Link from source content that's used in multiple projects](#link-from-source-content-thats-used-in-multiple-projects)
+- [Link to grafana.com pages](#link-to-grafanacom-pages)
+- [Link to external pages](#link-to-external-pages)
+- [Link to page headings](#link-to-page-headings)
+
+Other link types exist in our documentation, but you shouldn't use them:
+
+- [Hugo `relref` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#relref)
+
+## Link from source content that's used in multiple projects
+
+Use the `docs/reference` shortcode.
+
+The source is reused as described in [Reuse directories of content with Hugo mounts](https://grafana.com/docs/writers-toolkit/write/reuse-content/reuse-directories/).
+For more information and examples, refer to [`docs/reference` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsreference).
+
+## Link to grafana.com pages
+
+Use a fully qualified URL with version substitution syntax (if needed).
+Version substitution is necessary for fully qualified URLs to link to the correct version of documentation.
```

Comment:
```suggestion
Version substitution is necessary for full URLs to link to the correct version of documentation.
```

Reply:
If _qualified_ is important enough, can we define it in one place and then not use it in multiple places? It is hard to digest this particular adverb and adjective combination. Alternatively, how about "... a qualified URL" is we must know what _qualified_ means?

Reply:
Makes me think that I need to understand what a "half-qualified URL" might be.